### DISCUSSION TITLE
Ideas for Tasks in milestone 2 (Database Design)

### DISCUSSION THREAD

**Discussion by YampierFernandez** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#105] [Ideas]
Here we can post rough ideas for our team’s tasks, please speak your minds and remember to direct the ideas to the goals of the second milestone as much as possible:

As a reminder here is description of what we hope to focus on currently:

“The database team’s tasks will focus on connecting to the data to the backend, making sure the database can be populated, accessed and edited from the application instead of only through Atlas. The application should be able to perform functions such as get, pull, post, delete, etc. The necessary extensions to make this possible must applied and documented. The models will also be updated to better mix with the current needs of the application.

Many tasks will have to be done together with the backend team. Task that are better suited for both backend and database team members must be detailed as such so managers can know to assign members from both teams.”

**Comment by MarkoSchuetz** [comment]
I very much recommend starting from features that represent end-user value and to keep that as a guiding "light". Then better pare down the feature to the minimum (but still providing end-user value) then to split it along technological lines (database,...)
